spec/requests/portfolio_items_spec.rb

```diff
@@ -255,30 +255,30 @@
 
     context "when passing in read-only attributes" do
       before do
-        patch "#{api}/portfolio_items/#{portfolio_item.id}", :params => invalid_attributes, :headers => default_headers
+        patch "#{api}/portfolio_items/#{portfolio_item.id}", :params => invalid_attributes, :headers => default_headers, :as => :json
       end
 
-      it 'returns a 200' do
+      xit 'returns a 200' do
```

Comment:
@syncrou This is handled by the openapi validator it will prevent users from editing read only properties

Reply:
Should we simply be removing these specs then?

Reply:
@eclarizio I will be fixing it once I get the openapi validator and change the http_status_code to 400 (bad_request)